zeppelin-web/src/app/helium/helium.controller.js

```diff
@@ -24,6 +25,16 @@ function HeliumCtrl($scope, $rootScope, $sce, baseUrlSrv, ngToast, heliumService
   $scope.showVersions = {};
   $scope.bundleOrder = [];
   $scope.bundleOrderChanged = false;
+  $scope.vizTypePkg = {}
+  $scope.spellTypePkg = {}
+  $scope.intpTypePkg = {}
+  $scope.appTypePkg = {}
+  $scope.numberOfEachPackageByType = {}
+  
+  $scope.allPackageTypes = [HeliumType][0]
+  $scope.pkgListByType = 'VISUALIZATION'
```

Comment:
Then all interpreters registered in Maven central (about 33 artifacts) also will be shown mixed with all visualization and spell type of packages. I think this is a bit overdoing since currently we can't enable/disable the interpreters in Helium menu. How about showing all packages except `INTERPRETER` type? Not sure which way is better. Please let me know if you have better idea :)